features/step_definitions/town_steps.rb

```diff
@@ -61,6 +66,11 @@
 end
 
 
+Then(/^my repo is configured with the pull branch strategy "(.+?)"$/) do |pull_branch_strategy|
```

Comment:
Let's rename this step to "my repo is **now** configured with the pull branch strategy" to (a) imply that this step verifies existing state, and (b) distinguish it from "my repository has the "([^"]*)" pull branch strategy configured".